packages/service/fixtures/acme-base-ts/src/lib/index.ts

```diff
@@ -1,116 +0,0 @@
-import fp from 'fastify-plugin'
```

Comment:
Why was this removed? This test verified that substackables (like pg-hooks, ai-warp and others) would keep working. We need to provide a migration path for them.

Reply:
I moved these files. Anyway I accidentally deleted the embedded tests.
These are now restored and adapted. Will be in the next push.